lms/envs/aws.py

```diff
@@ -696,6 +705,9 @@
 FIELD_OVERRIDE_PROVIDERS += (
```

Comment:
Is there a concern where data retrieved from the [bulk operations](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/module_render.py#L145-L148) is used directly without user context binding? From my brief look, it doesn't look like this point is concerning